sbin/submitty_autograding_shipper.py

```diff
@@ -689,6 +689,10 @@ def get_job(my_name,which_machine,my_capabilities,which_untrusted,overall_lock):
         # skip items that are already being graded
         if (just_file[0:8]=="GRADING_"):
             continue
+
+        if (just_file[0:5]=="VCS__"):
```

Comment:
can use [startswith method](https://docs.python.org/3/library/stdtypes.html#str.startswith) (`just_file.startswith('VCS__')`) to be a bit more pythonic, otherwise looks fine.